src/Components/Profile/index.js

```diff
@@ -0,0 +1,32 @@
+import React from "react";
```

Comment:
Used react well and integrated tailwind efficiently enough to produce both a desktop and mobile app 👏